environments/development/base/outputs.tf

```
@@ -0,0 +1,19 @@
+output "region" {
+  description = "AWS region"
+  value       = var.region
+}
+
+output "public_subnet_id" {
+  description = "subnet id"
+  value       = module.vpc.public_subnets[*]
+}
+
+output "private_subnets_id" {
+  description = "subnet id"
+  value       = module.vpc.private_subnets[*]
+}
```

Similarly, here, we can do the same as the public subnets;

```suggestion
output "private_subnet_ids" {
  description = "List of private subnet IDs"
  value       = module.vpc.private_subnets
}
```